uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -0,0 +1,84 @@
+"""Tests related to uaclient.entitlement.base module."""
+
+import mock
+
+from uaclient import config
+from uaclient.entitlements.livepatch import LivepatchEntitlement
+from uaclient import status
+from uaclient.testing.helpers import TestCase
+
+
+class TestLivepatchContractStatus(TestCase):
+
+    def test_contract_status_entitled(self):
+        """The contract_status returns ENTITLED when entitled is True."""
+        tmp_dir = self.tmp_dir()
+        cfg = config.UAConfig(cfg={'data_dir': tmp_dir})
+        machineToken = {
```

Comment:
This looks like it's the same in every test; could we pull it out to a shared variable somewhere? (If we don't expect it to be mutated by the tests or the code, we could also make it a [MappingProxyType](https://docs.python.org/3/library/types.html#types.MappingProxyType) to ensure that they don't.)

Reply:
+1 will address this with a shared var and use deepcopy if we need to manipulate it.